src/app/GitExtUtils/GitUI/ComboBoxExtensions.cs

```diff
@@ -22,13 +25,23 @@ public static void AdjustWidthToFitContent(this ComboBox comboBox)
         ///  Resizes the drop-down width of a <see cref="ComboBox"/> to fit within provided bounds.
         /// </summary>
         /// <param name="comboBox">Target control</param>
-        /// <param name="minWidth">Minimum drop-down width</param>
-        /// <param name="maxWidth">Maximum drop-down width</param>
+        /// <param name="minWidth">Minimum drop-down width. Defaults to <see cref="ComboBoxExtensions.BranchDropDownMinWidth"/>.</param>
+        /// <param name="maxWidth">Maximum drop-down width. Defaults to <see cref="ComboBoxExtensions.BranchDropDownMaxWidth"/>.</param>
         /// <param name="dpiScaleBounds">Whether to apply Dpi scaling to <paramref name="minWidth"/> and <paramref name="maxWidth"/>. Defaults to <c>true</c>.</param>
-        public static void ResizeDropDownWidth(this ComboBox comboBox, int minWidth, int maxWidth, bool dpiScaleBounds = true)
+        public static void ResizeDropDownWidth(this ComboBox comboBox, int minWidth = -1, int maxWidth = -1, bool dpiScaleBounds = true)
```

Comment:
The compiler accepts:
```suggestion
       public static void ResizeDropDownWidth(this ComboBox comboBox, int minWidth = BranchDropDownMinWidth, int maxWidth = BranchDropDownMaxWidth, bool dpiScaleBounds = true)
```

Reply:
Done. They were `static readonly` when I wrote this part.. only later I've changed them to `const`.. sigh..